eclipse-scout-core/src/security/AccessControl.ts

```diff
@@ -52,9 +54,11 @@ export class AccessControl extends PropertyEventEmitter implements AccessControl
     const retryIntervals = [];
     if (interval) {
       let retryInterval = 1000;
-      while (retryInterval < interval) {
+      let factor = 2;
+      let limit = interval / factor; // retryInterval is the time between retries
+      while (retryInterval < limit) {
         retryIntervals.push(retryInterval);
-        retryInterval = retryInterval * 2;
```

Comment:
Doubling the retryInterval is probably not ideal, because it will soon be a very large interval. I know, it is nothing new, but I think we should change it. Maybe we have to adjust Call.ts to support this requirement, e.g. we could set maxRetries to -1 which means no limit and update this member based on the retryInterval only if maxRetries is null